src/Pattern.re

```diff
@@ -0,0 +1,43 @@
+module Tile = {
+  type t = array(array(int));
+
+  let from_bytes = (bytes: bytes, offset: int) => {
```

Comment:
This is nice. I'm sure I'm going to want to try to refactor it at some point anyway though. How do you make code like this look pretty and obvious _without_ having to refer to a PPU data structure reference first? Is such a thing possible? 🙃